Remove non-UTF-8 chars to fix build failure with latest glib-mkenums
--- libgnome/gnome-config.h.orig	2009-04-23 04:29:13.000000000 -0500
+++ libgnome/gnome-config.h	2017-10-04 02:17:22.000000000 -0500
@@ -270,7 +270,7 @@
 #define gnome_config_private_clean_key(path) \
 	(gnome_config_clean_key_((path),TRUE))
 
-/* returns the true filename of the config file */
+/* returns the true filename of the config file */
 #define gnome_config_get_real_path(path) \
 	(g_build_filename (gnome_user_dir_get(),(path),NULL))
 #define gnome_config_private_get_real_path(path) \
